packages/core/src/index.ts

```diff
@@ -210,6 +214,37 @@ export const create = (config: Config): VoiceCompass => {
 
   resetCallTimeout();
 
+  interface EscalationButtonProps {
+    wrap: HTMLElement | string,
+    text: string,
+  }
+
+  const appendEscalationButton = ({ wrap, text }: EscalationButtonProps) => {
```

Comment:
For completeness, we will need a `removeEscalationButton` method that disconnects the event listener and removes the button DOM element. Might want to keep a reference to the click handler and use `addEventListener` instead.